tools/cold-backup/README.md

```diff
@@ -1,26 +1,32 @@
-## Strimzi backup
+## Cold backup
 
-Bash script for offline (_cold_) backups of Kafka clusters on Kubernetes or OpenShift. 
-The script only supports a local file system, and you need to make sure to have enough free disk space in the target directory.
+Bash script for running cold backups of test or development Kafka clusters deployed on Kubernetes.
 
-If you think you don't need a backup strategy for Kafka because of its embedded data replication, then consider the impact of a misconfiguration, bug, or security breach that deletes all of your data. 
-For online (_hot_) backups, you can use storage snapshotting or stream into object storage.
+The backup procedure will stop the cluster operator and selected cluster for the entire duration of the backup.
```

Comment:
Should you pause it instead to avoid disruption of other clusters?

Reply:
Yes, that's actually what it does now. I just forgot to remove that sentence.